awx/requirements.lock

```diff
@@ -1,3 +1,9 @@
-dependencies: []
-digest: sha256:81059fe6210ccee4e3349c0f34c12d180f995150128a913d63b65b7937c6b152
-generated: 2019-10-01T15:47:32.476216-04:00
+dependencies:
+- name: redis
+  repository: https://charts.bitnami.com/bitnami
+  version: 14.4.0
+- name: postgresql
+  repository: ""
```

Comment:
Is this `repository` value supposed to be empty?

Reply:
yes, because we commit the chart locally instead of allowing it to download / install from the repository. Here's the line in the helm output that describes this behavior:

```
Dependency postgresql did not declare a repository. Assuming it exists in the charts directory
```